src/app/auth/components/registration/components/register/register.component.html

```diff
@@ -70,7 +111,7 @@
       <a class="links pantry-link" (click)="handlePantry()">Food Pantries Near Me</a>
     </div>
 
-    <p class=" vertical-rhythm-two text-primary text-center">
+    <p class=" verticalnpm start-rhythm-two text-primary text-center">
```

Comment:
verticalnpm start-rhythm-two. Non breaking but also not the right thing haha

Reply:
fixed